app/views/manuals/_content_for_manual.html.erb

```diff
@@ -1,6 +1,6 @@
 <% content_for :extra_headers do %>
-  <% if presented_manual.summary.present? %>
-    <meta name='description' content='<%= presented_manual.summary %>' />
+  <% if description.present? %>
+    <meta name='description' content='<%= description %>' />
```

Comment:
@h-lame that makes sense. I've added a few more commits (including renaming that partial).

What do you think?